### ISSUE TITLE
Add ability to select stylesheets on the fly

### ISSUE THREAD

**Issue by MichaelHilton** [hazelgrove/HZ#12]
this is lower priority, only once the other features have been implemented.

**Comment by ivoysey** [comment]
1. style that adds the triangle cursors back in, to really match the paper even at the expense of readability
2. style that shows the derivation for each action on the side? how hard would it be to compute a PNG of the derivation tree of what you just did via TeX?
